src/cross_section.F90

```diff
@@ -133,6 +142,15 @@ subroutine calculate_xs(p)
       ! Add contributions to material macroscopic energy release from fission
       material_xs % kappa_fission = material_xs % kappa_fission + &
            atom_density * micro_xs(i_nuclide) % kappa_fission
+
+      ! Add contributions to material macroscopic delayed-nu-fission cross
+      ! section
+      do d = 1, nuclides(i_nuclide) % n_precursor
+        yield = yield_delayed(nuc, p % E, d)
+        material_xs % delayed_nu_fission(d) = &
+             material_xs % delayed_nu_fission(d) +&
+             atom_density * micro_xs(i_nuclide) % delayed_nu_fission * yield
+      end do
```

Comment:
Okay, I'll make functions to compute the micro/macro delayed-nu-fission cross sections.